app/scripts/utils/interval-tree.js

```diff
@@ -167,7 +167,7 @@ function heightHelper(node) {
    * @method
    * @return {Number} Height of the tree.
    */
-IntervalTree.prototype.height = function () {
+IntervalTree.prototype.height = function height() {
```

Comment:
Why does this not have an underscore when the others do?

Reply:
The ones with underscores were failing lint because they shadowed functions with the same names and this seemed good enough, considering that nothing needs to actually reference these names; they're just for debugging.